read211_tests.py

```diff
@@ -61,12 +65,15 @@ def validate(filename, schema, instance):
 
 
 class TestStd211Translation(unittest.TestCase):
+    def setUp(self):
+        self.maxDiff = None
+
     def test_main_style_call(self):
         for file in test_files:
             warnings.simplefilter("ignore")
             wb = loadxl.load_workbook(file)
             warnings.simplefilter("default")
-            std211 = read211.read_std211_xls(wb)
+            std211 = read211.read_std211_xlsx(wb)
```

Comment:
does this mean it won't work with xls files?

Reply:
@nllong As far as I know, yes. There's an older python module that will apparently read xls files, but I don't believe that openpyxl will read those. And the code to extract all of the checkboxes is certainly xlsx only. If there's a need to read those files, there isn't a great solution other than converting the files to the newer xlsx format.